src/app/app.module.ts

```diff
@@ -28,13 +28,14 @@ import {NotFoundComponent} from './components/general/not-found/not-found.compon
 import {ForbiddenComponent} from './components/general/forbidden/forbidden.component'
 import {AdminComponent} from './admin/admin.component'
 import {AdminModule} from '@app/admin/admin.module'
+import {MyStatsComponent} from "@app/components/my-stats/my-stats.component"
 
 import {
     TUI_SANITIZER,
     TuiButtonModule,
     TuiDataListModule,
     TuiDialogModule,
-    TuiDropdownControllerModule,
+    TuiDropdownControllerModule, TuiFormatNumberPipeModule,
```

Comment:
These should be in 2 different lines. Have you run `npm run lint`? I think it will catch these formatting issues.